extensions/icons/package.json

```
@@ -0,0 +1,42 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "icons",
```

I would rename the extension and the folder to `iconify` 
```suggestion
  "name": "iconify",
```